interop_task/interop_task.md

```diff
@@ -0,0 +1,124 @@
+| Proposal ID | TBC |
+|-------------|--------|
+| Name |  |
+| Date of Creation | 16 January 2019 |
+| Target | SYCL 1.2.1 extension |
+| Current Status | _Work in progress_ |
+| Reply-to | Victor Lomüller <victor@codeplay.com> |
+| Original author | Victor Lomüller <victor@codeplay.com>, Gordon Brown <gordon@codeplay.com>, Peter Zuzek <peter@codeplay.com> |
+| Contributors | Victor Lomüller <victor@codeplay.com>, Gordon Brown <gordon@codeplay.com>, Peter Zuzek <peter@codeplay.com> |
+
+# interop_task: Improving SYCL-OpenCL Interoperability
+
+## Motivation
+
+SYCL does not allow a user to access cl_mem object out of an cl::sycl::accessor, it difficult/impossible to integrate OpenCL library to use them as is inside the data-flow execution model of SYCL, as the only current way to do this is to create all OpenCL buffers up-front, which is not always possible.
```

Comment:
it difficult/impossible -> It is difficult to

integrate OpenCL library to use them as is inside -> integrate an OpenCL library seamesly inside the data-flow execution model of SYCL

Reply:
I'll replace "OpenCL" by "low-level API functionality" to be consistent with previous changes